src/omotes_sdk_protocol/workflow.proto

```diff
@@ -51,10 +51,25 @@ message WorkflowParameter {
   }
 }
 
+message ParameterRelation {
```

Comment:
See comments by Cheng-Kai and myself: https://github.com/Project-OMOTES/orchestrator/pull/83#pullrequestreview-2502537374

Suggestion to rename to `ParameterConstraint` and only have 1 `other_parameter`.

Reply:
or `ref_parameter_key_name` (more explicit)